contents/dataframes_pt.md

```diff
@@ -21,15 +21,15 @@ Options(tv_shows; label="TV_shows")
 ```
 
 Aqui, as reticências significam que esta pode ser uma tabela muito longa e mostramos apenas algumas linhas.
-Ao analisar dados, muitas vezes levantamos questões interessantes sobre eles.
+Ao analisar dados, muitas vezes levantamos questões interessantes sobre eles, também chamadas de __queries (ou consultas) de dados__.
```

Comment:
```suggestion
Ao analisar dados, muitas vezes levantamos questões interessantes sobre eles, também chamadas de _queries_ (ou consultas) de dados.
```
Usamos `_` para itálico e `**` para negrito.
Veja se era isso que você tinha intenção.